README.md

```diff
@@ -21,9 +21,15 @@ this is `sudo dnf install libudev-devel`).
 #### TL;DR
 
 ```bash
+# Make sure to update & initialize submodules
+git submodule update --init --recursive
+
 # Set up and compile
 scons
 
+# To build fw
+scons fw
```

Comment:
Could you replace the `scons` line above with `scons fw` instead?

Reply:
done.